report-viewer/package.json

```diff
@@ -0,0 +1,52 @@
+{
```

Comment:
You may want to add node/npm to the dependabot.yml to ensure updates.

Reply:
Also: Please ensure that you use current versions in your `package.json`. The Installation already yields several vulnerabilities:
```
found 30 vulnerabilities (2 low, 22 moderate, 6 high)
  run `npm audit fix` to fix them, or `npm audit` for details
```